README.rst

```diff
@@ -26,105 +26,48 @@ Molecule
    :alt: Repository License
 
 Molecule is designed to aid in the development and testing of `Ansible`_ roles.
+
 Molecule provides support for testing with multiple instances, operating
 systems and distributions, virtualization providers, test frameworks and
-testing scenarios.  Molecule is opinionated in order to encourage an approach
-that results in consistently developed roles that are well-written, easily
-understood and maintained.
-
-Molecule uses `Ansible`_ `playbooks`_ to exercise the `role`_ and its
-associated tests.  Molecule supports any provider [#]_ that `Ansible`_
-supports.
-
-.. [#]
-
-   Providers can be bare-metal, virtual, cloud or containers.  If Ansible can
-   use it, Molecule can test it.  Molecule simply leverages Ansible's module
-   system to manage instances.
-
-.. _`playbooks`: https://docs.ansible.com/ansible/latest/playbooks.html
-.. _`role`: https://docs.ansible.com/ansible/latest/playbooks_roles.html
-
-Quick Start
```

Comment:
@decentral1se do you intend to completely remove this from README (which would include GitHub repo view and PyPI page)?

There's a way to keep this in one source file (README) and then include parts of readme into different pages on RTD using `include` SphinX directive with `start-after`/`end-before` markers.

Example: https://github.com/ansible/ansible-lint/pull/420/files#diff-3caecf318f72e748f256c37866cc61f4R13

Reply:
Awesome! That was much needed, I rigged it up in https://github.com/ansible/molecule/pull/1776/commits/d740306d5302be04b3d4711d8df4afce4e531d98.